modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/entity/BlazeAd.kt

```diff
@@ -0,0 +1,17 @@
+package au.com.shiftyjelly.pocketcasts.models.entity
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import au.com.shiftyjelly.pocketcasts.models.type.BlazeAdLocation
+import java.io.Serializable
+
+@Entity(tableName = "blaze_ads")
+data class BlazeAd(
+    @PrimaryKey(autoGenerate = false) @ColumnInfo(name = "id") var id: String,
+    @ColumnInfo(name = "text") var text: String,
+    @ColumnInfo(name = "image_url") var imageUrl: String,
+    @ColumnInfo(name = "url_title") var urlTitle: String,
+    @ColumnInfo(name = "url") var url: String,
+    @ColumnInfo(name = "location") var location: BlazeAdLocation,
+) : Serializable
```

Comment:
ℹ️ There's no need to implement `Serializable` interface.

Reply:
I have this error if I remove the `Serializable`. Could I get some advice, should I be making it Parcelable?

> e: pocket-casts-android/modules/features/ads/src/main/kotlin/au/com/shiftyjelly/pocketcasts/ads/AdReportFragment.kt:92:17 Type is not directly supported by 'Parcelize'. Annotate the parameter type with '@RawValue' if you want it to be serialized using 'writeValue()'.

```
    private class Args(
        val ad: BlazeAd,
        val podcastColors: PodcastColors?,
    ) : Parcelable
```